### PR TITLE
Added documentation on newly added CMake support for samples

### LINKED ISSUE #1632: OE samples readme files should be updated to reflect the newly added CMake support

### PR DESCRIPTION
**Pull request by soccerGB**
Fixes #1632.

### CODE REVIEW

File: samples/data-sealing/README.md [hunk 1]
```diff
@@ -157,7 +154,7 @@ cmake ..
 make run
 ```
 
-### pkg-config
+### GNU CMake with pkg-config
```
Comment (andschwa) [inline samples/data-sealing/README.md]: I think you mean "GNU Make" 😉

File: samples/README.md [hunk 2]
```diff
@@ -69,23 +69,44 @@ Package oeenclave-clang was not found in the pkg-config search path.
...
 For example:
 
 ```bash
 username@yourVMname:~/openenclave/share/openenclave/samples$ cd helloworld/
-username@yourVMname:~/openenclave/share/openenclave/samples/helloworld$ ls enc  host  Makefile
-
 username@yourVMname:~/openenclave/share/openenclave/samples/helloworld$ make build
- ...
-username@yourVMname:~/openenclave/share/openenclave/samples/helloworld$ make run host/helloworldhost ./enc/helloworldenc.signed
+username@yourVMname:~/openenclave/share/openenclave/samples/helloworld$ make run
```
Comment (andschwa) [inline samples/README.md]: @CodeMonkeyLeet had informed me that we should remove the prompts from sample shell code to keep them simple, but I don't recall what, precisely, the format should be.

File: samples/helloworld/README.md [hunk 3]
```diff
@@ -492,22 +492,31 @@ The following files are generated during the build.
...
 ```bash
-./host/helloworldhost ./enc/helloworldenc.signed
+cd helloworld
+mkdir build && cd build
+cmake ..
+make run
 ```
 
-Or execute `make run` from the root of the sample:
+### GNU CMake
```
Comment (andschwa) [inline samples/helloworld/README.md]: Ditto

File: samples/file-encryptor/README.md [hunk 4]
```diff
@@ -132,12 +132,24 @@ Free all the resources allocated for this encryptor instance.
...
+This uses the CMake package provided by the Open Enclave SDK.
+
+```bash
+cd file-encryptor
+mkdir build && cd build
+cmake ..
+make run
+```
+
+### GNU CMake
```
Comment (andschwa) [inline samples/file-encryptor/README.md]: Ditto

File: samples/local_attestation/README.md [hunk 5]
```diff
@@ -226,12 +226,24 @@ See [here](https://github.com/Microsoft/openenclave/tree/master/docs/MbedtlsSupp
...
+This uses the CMake package provided by the Open Enclave SDK.
+
+```bash
+cd local_attestation
+mkdir build && cd build
+cmake ..
+make run
+```
+
+### GNU CMake
```
Comment (andschwa) [inline samples/local_attestation/README.md]: Ditto (and I don't' know why this one is in italics, lol GitHub)

**Comment by johnkord** [comment]
@soccerGB @andschwa What's the status of this PR? Are we going to merge this soon?

File: samples/local_attestation/README.md (hunk 5 above)
Comment (soccerGB) [reply samples/local_attestation/README.md]: addressed

File: samples/data-sealing/README.md (hunk 1 above)
Comment (soccerGB) [reply samples/data-sealing/README.md]: addressed

File: samples/README.md (hunk 2 above)
Comment (soccerGB) [reply samples/README.md]: addressed

File: samples/helloworld/README.md (hunk 3 above)
Comment (soccerGB) [reply samples/helloworld/README.md]: addressed

File: samples/file-encryptor/README.md (hunk 4 above)
Comment (soccerGB) [reply samples/file-encryptor/README.md]: addressed

**Comment by andschwa** [comment]
Flaky test.

bors r+

**Comment by andschwa** [comment]
See #1659

**Pull request merged**